rocAL/rocAL/CMakeLists.txt

```diff
@@ -268,4 +282,4 @@ if(${BUILD_ROCAL})
     endif(BUILD_DEV)
 else()
     message("-- ${Red}WARNING: rocAL module excluded ${ColourReset}")
-endif()
+endif()
```

Comment:
Please revert this change and add line at EOF.

Reply:
Added EOF - reverted the change